lib/win32/mixin/crypto.rb

```diff
@@ -0,0 +1,109 @@
+#
+# Author:: Nimisha Sharad (<nimisha.sharad@msystechnologies.com>)
+# Copyright:: Copyright (c) 2017 Chef Software, Inc.
+# License:: Apache License, Version 2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+require 'ffi'
+require 'chef'
+require 'chef/win32/api'
+require 'mixlib/shellout'
+
+module Win32
+  module Mixin
```

Comment:
can you move this to `Win32::Crypto::Mixin` please? one more directory. Nothing here is meant to be shared with other Win32 libraries so let's use our own namespace.